setup.py

```diff
@@ -194,7 +194,7 @@ def extract_version():
 
 
 setup(
-    name='Iris',
+    name='iris-scitools',
```

Comment:
I think I prefer iris-scitools.  If I was searching for this, I would feel more like I had found what I was looking for this way round than the other way around.